libs/interpret-vision/src/lib/VisionExplanationDashboard/Controls/FlyoutObjectDetection.tsx

```diff
@@ -103,9 +100,13 @@ export class FlyoutObjectDetection extends React.Component<
       });
     }
   }
-
+  public callbackWrapper = (): void => {
```

Comment:
can this be made a private function instead?  it doesn't seem like it needs to be public since it's not called from another file but just within this file?